doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,54 @@
+"""
+===============================================
+Reconstruct dust-covered image using inpainting
+===============================================
+
+It is possible that dust gets accumulated on the reference mirror and causes
+dark spots to appear on direct images. This example reproduces the steps taken
+to perform OCT (Optical Coherence Tomography [1]_) dust removal in images.
+This application was first discussed by Jules Scholler in [2]_.
+
+.. [1] Vinay A. Shah M.D. (2015)
+       `Optical Coherence Tomography <https://eyewiki.aao.org/Optical_Coherence_Tomography#:~:text=3%20Limitations-,Overview,at%20least%2010%2D15%20microns.>`_,
+       American Academy of Ophthalmology.
+.. [2] Jules Scholler (2019) "Image denoising using inpainting":
+       `<https://www.jscholler.com/2019-02-28-remove-dots/>`_
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import imageio.v3 as iio
+import plotly.express as px
+
+
+#################################################################################
+# The dataset that we are using in this example is an image sequence showing
+# the palisades of Vogt in vivo in a human cornea. The file has been
+# acquired in TIFF format.
+
+raw = iio.imread('data/raw.tiff')
```

Comment:
```suggestion
image_sequence = iio.imread('data/raw.tiff')
```
Although, in light of this [discussion](https://github.com/datacarpentry/image-processing/issues/258), maybe we should start using descriptive names...

Reply:
That improves the accessibility of our tutorials! Although, are we also trying to limit the size of variables to avoid clutter? Because if we replace `raw` ---> `image_sequence`, do we also replace `raw_mean` ---> `image_sequence_mean`?

Reply:
> Although, are we also trying to limit the size of variable names to avoid clutter?

Yes and no... Explicit and descriptive are better (more educational).